test/FunctionalTestUtils/InProcessServer.cs

```diff
@@ -31,7 +31,8 @@ public BackTelemetryChannel BackChannel
         public InProcessServer(string assemblyName, Func<IWebHostBuilder, IWebHostBuilder> configureHost = null)
         {
             this.configureHost = configureHost;
-            var machineName = Environment.GetEnvironmentVariable("COMPUTERNAME");
+            //var machineName = Environment.GetEnvironmentVariable("COMPUTERNAME");
```

Comment:
worth leaving comment here why it's `localhost`. Also do not leave commented code

Reply:
commented.
linux cant get machines names easily. it was localhost always but removed to let X-componet header injection. Does not look like any tests are depending on x-component header injection.